### ISSUE TITLE
Userdefined Variations text gets capitalized in basket and emails

### ISSUE THREAD

**Issue by ajgw** [woocommerce/woocommerce#16791]
I have setup a store with userdefined variations as this kind:

30x40 cm | 40x50 cm and so on

If I put such a product variation into basket, the word cm which is a metric value gets capitalized and in emails too: cm gets Cm.

I tried this on two different sites, one with Storefront as theme. But it is always there.

It is not really a bug but it doesn't look well. Thanks.

Oops: It is a solution in 3.1.2!

**Comment by mikejolley** [comment]
> Oops: It is a solution in 3.1.2!

I assume you mean it was fixed in that version? We should no longer force caps on attribute names. Let me know if it's still a problem.

Thanks

**Comment by webbverkstaden** [comment]
@mikejolley Running 3.1.2, and this is still a problem.

**Comment by claudiosanches** [comment]
@webbverkstaden there is no code forcing caps anymore, so I bet that is probably something on your theme.

**Comment by webbverkstaden** [comment]
@claudiosanches Ok, drove me nuts. Turns out that I still have **ucwords** in woocommerce/includes/wc-product-functions.php:380
```php
// If this is a term slug, get the term's nice name
			if ( taxonomy_exists( $name ) ) {
				$term = get_term_by( 'slug', $value, $name );
				if ( ! is_wp_error( $term ) && ! empty( $term->name ) ) {
					$value = $term->name;
				}
			} else {
				$value = ucwords( str_replace( '-', ' ', $value ) );
			}
```
And I am on **3.1.2**. Grabbing a fresh copy of woocommerce from WP and the above code **is still there**. So something went sideways when you pushed this to WP.

Thanks anyway, case closed.

**Comment by ajgw** [comment]
@webbverkstaden This is solved in WC 3.2 of which we are talking. My first submit was related to 3.1.2.

So beware, in WC 3.2 it is fixed.
